integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/Hooks.java

```diff
@@ -40,6 +44,28 @@ public Hooks(TestContext context) {
     }
 
     @BeforeAll
+    public static void setUp() {
+        //Initialize env variables if Local config properties file exists for running tests locally
+        initializeLocalTestConfig();
+        //clear older Webhook site requests
+        webhookSiteRequestHistoryIsCleared();
+    }
+
+    public static void initializeLocalTestConfig() {
+        final String filePath = "src/test/resources/localconfig.properties";
```

Comment:
Using classpath location is unfortunate as it can lead to properties exposure (properties file is part of test jar). I think it will be better to place the `localconfig.properties` file directly into `integration-tests` module root.